config/common/errors.go

```diff
@@ -32,6 +32,7 @@ var (
 
 	// resources and trees
 	ErrTooManyResourceSources = errors.New("only one of the following can be set: inline, local, source")
+	ErrTooManySystemdSources  = errors.New("only one of the following can be set: inline, local")
```

Comment:
`contents` and `contents_inline`.  And maybe this shouldn't go in the resource/tree section, since it applies to different structures?